_sections/self-hosted-admin-android-studio.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Android Studio
+tab: android
+---
+
+### Overview
+
+In this quick guide you will learn how to install the GitLive app on your self hosted Gitlab instance as an administrator.
+
+### Steps for authenticating
+As an administrator, you are required to install the GitLive app on the group level and grant it access to the repositories that you would like to use GitLive with. Note that your instance will need to be accessible from the internet in order for the integration to work correctly.
+### Step 1
+* Install the plugin as explained in the [installation section](/){:class="internal-link"}.
+* Open up a clone of the repository that you want to use GitLive with.
+* Then, you will see the following prompt in the GitLive window:
+
+![Configure Server](/uploads/jetbrains-gsh-configure-server.jpg "Configure Server"){:class="screenshot"}
+
+
+### Step 2
+
+After clicking on "Configure new GitLab server", follow the setup instructions. You'll start by adding a new OAuth 2.0 application in GitLab, this will allow you to set the right scopes, and get the GitLab Application ID and Secret, which are needed in GitLive.
```

Comment:
"Configure new GitLab server". is there a better markup way then "". italic or something

Reply:
which you need for the set up